### PR TITLE
added swift lint to project

### PR DESCRIPTION
**Pull request by uixbuilder**

### CODE REVIEW

File: RunScripts/rdlint.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# PATH in new macOS versions doesn't include path below where rdlint is installed
+export PATH=$PATH:/usr/local/bin
+
+if which rdlint > /dev/null; then
+    rdlint swiftlint
+else
+    echo "error: rdlint is not installed. Follow instructions here: https://github.com/readdle/rdlint"
```
Comment (Artem-Shuliak) [inline RunScripts/rdlint.sh:9]: I don't have access to the repository with the RdLint. Can you please give me access to the repo, so I can install it, before running the script?

Comment (uixbuilder) [reply RunScripts/rdlint.sh:9]: Sure, I'm working on it )

Comment (uixbuilder) [reply RunScripts/rdlint.sh:9]: done, please check the link again

**Review by Artem-Shuliak — APPROVED** [review]
Looks Good!

**Pull request merged**